chapters/ru/_toctree.yml

```diff
@@ -3,11 +3,23 @@
   - local: chapter0/1
     title: Введение
 
-- title: 1. Transformer models
+- title: 1. Трансформеры
   sections:
   - local: chapter1/1
     title: Введение
   - local: chapter1/2
     title: Обработка естесственного языка
   - local: chapter1/3
     title: Трансформеры, на что они способны?
+  - local: chapter1/4
+    title: Как работают трансформеры?
+  - local: chapter1/5
+    title: Модели энкодеров
+  - local: chapter1/6
+    title: Модели декодеров
+  - local: chapter1/7
+    title: Модели "seq2seq"
+  - local: chapter1/8
+    title: Предвзятости и ограничения
+  - local: chapter1/9
```

Comment:
> Did you wish to include the quiz in this PR or leave that for another time / someone else? No need to include it, just asking in case you didn't see it

For now, I'd rather leave it for the future. Thank you!

Reply:
Fine with me, thanks!